README.md

```diff
@@ -389,6 +392,7 @@ Options
   --tsconfig            Specify your custom tsconfig path (default <root-folder>/tsconfig.json)
   --verbose             Keep outdated console output in watch mode instead of clearing the screen
   --noClean             Don't clean the dist folder
+  -T, --transpileOnly   Skip type checking
```

Comment:
Can we omit the short hand actually? We don't use it on others aside from help. I think we should open a new issue / pr and do them all at once.

Reply:
Of course, my idea was to use the same args as `ts-node` so people feel familiar 🙂

Reply:
PS: will change it tomorrow (it's late in 🇩🇪)